.github/blueprints/blueprint.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+  "landingPage": "/wp-admin/plugins.php",
```

Comment:
I think in order for this to be a proper demo, we need to do a few more things.

1. Maybe import some content
2. Generate the sitemap
3. Flush rewrites
4. Send the user to the sitemap.xml page